components/grid/export/excel.md

```diff
@@ -99,8 +103,10 @@ Optionally, you can also set the `GridExcelExport` tag settings under the `GridE
 
 You can programmatically invoke the export feature of the Grid, by using the following methods exposed on the `@ref` of the Grid:
 
-* `SaveAsExcelFileAsync` - `ValueTask` - sends the exported excel file to the browser for download.
-* `ExportToExcelAsync` - `Task<MemoryStream>` - returns the exported data as a `MemoryStream`. The stream itself is finalized, so that the resource does not leak. To read and work with the stream, clone its available binary data to a new `MemoryStream` instance.
+| Method | Type | Description |
+| --- | --- | --- |
+| `SaveAsExcelFileAsync` | `ValueTask` | sends the exported excel file to the browser for download. |
+| `ExportToExcelAsync` | `Task<MemoryStream>` | returns the exported data as a `MemoryStream`. The stream itself is finalized, so that the resource does not leak. To read and work with the stream, clone its available binary data to a new `MemoryStream` instance. |
```

Comment:
```suggestion
| `ExportToExcelAsync` | `Task<MemoryStream>` | Returns the exported data as a `MemoryStream`. The stream itself is finalized, so that the resource does not leak. To read and work with the stream, clone its available binary data to a new `MemoryStream` instance. |
```